docs/source/api.rst

```diff
@@ -471,3 +471,32 @@ Dask Base Classes and Mixins
 
 .. autoclass:: cuml.dask.common.base.DelayedInverseTransformMixin
    :members:
+
+Experimental
+============
+
+.. warning:: The `cuml.experimental` module contains features that are still
+    under development. It is not recommended to depend on features in this
+    module as they may change in future releases.
+
+.. note:: Due to the nature of this module, it is not imported by default by
+    the root `cuml` package. Each `experimental` submodule must be imported
+    separately.
+
+Decomposition
+-------------
+.. autoclass:: cuml.experimental.decomposition.IncrementalPCA
+   :members:
+
+HyperOpt Utilities
```

Comment:
I think we'll deprecate this hyperopt bit in the future and move that all to cloud-ml-examples, so I would just leave it out.

Reply:
> Also it would be nice to check that no documentation is generated for `sklearn` preprocessing models that are not yet incorporated into `cuML`.

I only included the classes and functions that were listed in `python/cuml/experimental/preprocessing/__init__.py`. I assumed these were the only ones that were customized and should be included. Here is the full list:

```
# Classes
'Binarizer',
'KBinsDiscretizer',
'MaxAbsScaler',
'MinMaxScaler',
'Normalizer',
'PolynomialFeatures',
'RobustScaler',
'SimpleImputer',
'StandardScaler',
# Functions
'add_dummy_feature',
'binarize',
'minmax_scale',
'normalize',
'robust_scale',
'scale',
```

Reply:
Removed the HyperOpt Section